docs/coronavirus_page_publishing_tool.md

```diff
@@ -0,0 +1,48 @@
+## Coronavirus Publishing Tool
+
+There are currently three coronavirus hub pages(/coronavirus/business-support, /coronavirus/education-and-childcare, /coronavirus/worker-support), and a coronavirus landing page. As we approached lockdown, a basic publishing interface was required at short notice. This is how it works:
+
+1. Content is added to the relevant yaml file in the [govuk-coronavirus-content](https://github.com/alphagov/govuk-coronavirus-content/tree/master/content) repository.
+2. Content editor with Coronavirus editor signon permissions visits [Coronavirus pages](https://collections-publisher.publishing.service.gov.uk/coronavirus) tab of collections publisher.
+3. Selects which page to publish.
+4. Clicking update draft and publish buttons, fetches the yaml file, parses it, and sends it to publishing api as JSON.
+5. Collections renders the page by reading from this updated content item.
+
+### First iteration of new tool
+
+In order to reduce the load on content designers and supporting developers, a better publishing tool was required that would remove the need to make content changes via github. The first iteration of this tool was to move the accordion content out of the yaml file in govuk-coronavirus-content, and into the collections publisher database. This means it can be edited from collections publisher directly via a friendlier UI. **NB the content for the rest of the page must still be edited via the steps above.**
```

Comment:
Perdantic point, but this paragraph starts in the past tense and then shifts to the present tense half way through. As I presume this is a live document, and the behaviour may well change, this should be a record of what happened at the first iteration and therefore should all be in the past tense - even where behaviour introduced in this iteration is still extant. So:
```
This means it could be edited from collections publisher directly via a friendlier UI. 
**NB the content for the rest of the page would still be edited via the steps above.**
```